flixel/graphics/FlxMaterial.hx

```diff
@@ -0,0 +1,279 @@
+package flixel.graphics;
+
+/*
```

Comment:
I'm working on next iteration of renderer which uses this class. It will support multitexture rendering. Hope to finish it soon